internal/pkg/logging/workload_test.go

```diff
@@ -165,12 +165,13 @@ firelens_log_router/fcfe4 10.0.0.00 - - [01/Jan/1970 01:01:01] "WARN some warnin
 		},
 		"success with state machine included": {
 			includeStateMachine: true,
+			last:                1,
 			setupMocks: func(m workloadLogsMocks) {
 				gomock.InOrder(
 					m.logGetter.EXPECT().LogEvents(gomock.Any()).
 						Do(func(param cloudwatchlogs.LogEventsOpts) {
 							require.Equal(t, param.LogStreamPrefixFilters, []string{"copilot/", "states"})
-							require.Equal(t, param.Limit, aws.Int64(10))
+							require.Equal(t, param.LogStreamLimit, 2)
```

Comment:
are we still interested in what's passed in `param.Limit`? If so, what do you think of
```go
require.Equal(t, param.LogStreamPrefixFilters, []string{"copilot/", "states"})
require.Equal(t, param,Limit aws.Int(10))
require.Equal(t, param.LogStreamLimit, 2)
```

Reply:
Good idea; i've added a check here and a new test to ensure that some log stream limit behavior is working properly.